workloads/whereabouts/metrics-profiles/hosted-cp-metrics.yml

```diff
@@ -0,0 +1,32 @@
+# Hosted control-plane metrics
```

Comment:
Do we care to have this? Not really testing HCP w/ Multus as of now, not sure we need to keep this.

Reply:
this too, shall leave